src/components/DocumentList.tsx

```
@@ -0,0 +1,135 @@
+  ModalBody,
+  HStack,
+} from "@chakra-ui/react";
+import { CheckIcon, CloseIcon, InfoOutlineIcon } from "@chakra-ui/icons";
+import Table from "./common/table/Table";
+
+// Document interface
+export interface Document {
+  id: number;
+  type: string;
+  title: string;
+  content: any;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)